ipatests/prci_definitions/gating.yaml

```diff
@@ -51,7 +51,7 @@ jobs:
         timeout: 3600
         topology: *master_1repl
 
-  fedora-27/external_ca:
+  fedora-27/test_external_ca_TestExternalCA:
```

Comment:
To keep a consistent name between the branches, I would rather use the name `fedora27/external_ca_1`.

Reply:
@flo-renaud here external_ca titles make bit more sense rather than using external_ca1 and external_ca2.